app/controllers/trainer_controller.rb

```diff
@@ -2,9 +2,9 @@ class TrainerController < ApplicationController
   respond_to :html
 
   def review
-    @card = Card.find(params[:id])
+    @card = current_user.cards.find(params[:id])
     if @card.check_translation(trainer_params[:user_translation])
-      flash[:alert] = 'Вы ввели верный перевод. Продолжайте.'
+      flash[:notice] = 'Вы ввели верный перевод. Продолжайте.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.